Snippets/Core/Core_6/Errors/SecondLevel/DisableWithCode.cs

```diff
@@ -1,14 +1,13 @@
 ﻿namespace Core6.Errors.SecondLevel
 {
     using NServiceBus;
-    using NServiceBus.Features;
 
     class DisableWithCode
     {
         DisableWithCode(EndpointConfiguration endpointConfiguration)
         {
             #region DisableSlrWithCode
-            endpointConfiguration.DisableFeature<SecondLevelRetries>();
+            endpointConfiguration.SecondLevelRetries().Disable();
```

Comment:
@Particular/nservicebus-maintainers 
Why is this inconsistent?
i.e. for FLR it's

> endpointConfiguration.DisableFirstLevelRetries();

Also what happens when you disable FLR but not SLR?